x-pack/plugins/security_solution/server/endpoint/schemas/artifacts/saved_objects.ts

```diff
@@ -77,3 +77,51 @@ export const internalManifestCreateSchema = t.intersection([
   ),
 ]);
 export type InternalManifestCreateSchema = t.TypeOf<typeof internalManifestCreateSchema>;
+
+/**
+ * Unified Manifest
+ */
+
+const internalUnifiedManifestBaseSchema = t.exact(
+  t.type({
+    artifactIds: t.array(identifier),
+    policyId: identifier,
+    semanticVersion,
+  })
+);
+export type InternalUnifiedManifestBaseSchema = t.TypeOf<typeof internalUnifiedManifestBaseSchema>;
+
+const internalUnifiedManifestCreateSchema = t.intersection([
+  internalUnifiedManifestBaseSchema,
+  t.type({
+    created,
+  }),
+]);
+
+export type InternalUnifiedManifestCreateSchema = t.TypeOf<
+  typeof internalUnifiedManifestCreateSchema
+>;
+
+const internalUnifiedManifestUpdateSchema = t.intersection([
+  internalUnifiedManifestBaseSchema,
+  t.exact(
+    t.type({
+      id: identifier,
+    })
+  ),
+]);
+export type InternalUnifiedManifestUpdateSchema = t.TypeOf<
+  typeof internalUnifiedManifestUpdateSchema
+>;
+
+export const internalUnifiedManifestSchema = t.intersection([
+  internalUnifiedManifestBaseSchema,
+  t.exact(
+    t.type({
+      id: identifier,
+      created,
+    })
+  ),
+]);
+
+export type InternalUnifiedManifestSchema = t.TypeOf<typeof internalUnifiedManifestSchema>;
```

Comment:
I followed the already existing pattern of typing for artifacts manifest. Do you see any reason why it should stay this way? If not I would go with TS interfaces/types instead.